docs/_docs/permalinks.md

```diff
@@ -83,19 +94,35 @@ Here's the full list of placeholders available:
       </td>
       <td>
         <p>
-          Month without leading zeros from the post's filename. May be
-          overridden via the document’s <code>date</code> front matter
+          Month without leading zeros from the post’s filename. May be
+          overridden via the document’s <code>date</code> front matter.
         </p>
       </td>
     </tr>
+    <tr>
+      <td>
+        <p><code>short_month</code></p>
+      </td>
+      <td>
+        <p>Three-letter month abbreviation, e.g. _JAN_</p>
```

Comment:
Since we're using HTML here. Markdown won't be processed, so this shows up in the docs like so:

![image](https://user-images.githubusercontent.com/77174/54497540-e2ab1500-48c9-11e9-9c2f-40c3c1d54b66.png)

Based on the code here, if you'd like them to be italicized, you'll want to use an `<i>` tag.
```suggestion
        <p>Three-letter month abbreviation, e.g. <i>JAN</i></p>
```